recipes/liknorm/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "liknorm" %}
+{% set version = "1.2.0" %}
+{% set sha256 = "be2953de3cdca93ec9964123db07436917265e226e08892ebab59f17ea6371ce" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/glimix/liknorm/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  features:
+    - vc14
+  skip: True  # [win32 or (win64 and py27 or win64 and py34)]
+
+requirements:
+  build:
+    - toolchain
+    - cmake
+
+test:
+  commands:
+    - test -e $PREFIX/lib/libliknorm.so  # [linux]
+    - test -e $PREFIX/lib/libliknorm.a  # [osx]
+    - test -e $PREFIX/lib/libliknorm.dylib  # [osx]
+
+about:
+  home: http://github.com/glimix/liknorm
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: 'C library for fast numerical integration'
```

Comment:
I think it would be more descriptive to use the following:

> C library for computing moments of the product of an exponential-family likelihood with a Normal distribution.

The current description makes it seem more general.